lib/framework/UploadFileService.cpp

```diff
@@ -143,9 +144,9 @@ void UploadFileService::handleError(AsyncWebServerRequest * request, int code) {
     }
 
     // send the error code to the client and record the error code in the temp object
-    request->_tempObject              = new int(code);
     AsyncWebServerResponse * response = request->beginResponse(code);
     request->send(response);
+    handleEarlyDisconnect();
```

Comment:
maybe here also `request->client()->close();` to stop the browser uploading. `handleError`is also called for wrong bin target, and uploading a bin takes a while.

Reply:
I wanted to send a response code back first, gracefully, but when calling client()->close() it disconnects quickly and the whole response is lost. So I had to work around it and catch the network error in the web code.

I also noticed the upload is slower than with axios. I'll log an issue with the alova team to see what is causing this, along with a benchmark.

Reply:
> and the whole response is lost

Seems my computer or network is to slow, i always get the response. I also dont see a slower upload speed. But for most people uploading is very rare, so it does not matter if it takes a few seconds more.